doc/policies.md

```diff
@@ -31,3 +31,55 @@ A release (identified by a unique semantic version) is protected against
 changes by its integrity hashes. If errors are identified post-publication, a
 release could be withdrawn, or superseded by a new one (using the appropriate
 major/minor/patch/build version changes), but not modified.
+
+## Best practices
+
+ - Avoid using `ada` as a prefix for your crate name, this will make the
+   project harder to find in a list. `ada` suffix is ok when the project is a
+   binding for an existing library (e.g. `sdlada`, `gtkada`).
+ 
+ - Split big projects in multiple crates:
+    
+    - If your project is a collection of components (like GNATcoll for
+      instance) and each component has different dependencies, you should
+      consider splitting the collection into multiple Alire crates. The
+      components can still be maintained in the same repository and use the
+      same release archive/commit (e.g.
+      [gnatcoll_sqlite](https://alire.ada.dev/crates/gnatcoll_sqlite),
+      [gnatcoll_sql](https://alire.ada.dev/crates/gnatcoll_sql),
+      [gnatcoll_postgres](https://alire.ada.dev/crates/gnatcoll_postgres)).
+    
+    - If your project is an application/executable/tool, some parts of the
+      application are potentially interesting on their own and would benefit
+      the ecosystem. For instance a parser for a standard file format.
```

Comment:
Something reads funky in this phrase. Is the intention to say that a library could be made from these parts?

Reply:
That is my indention, yes.